src/documentReview/Preview.vue

```diff
@@ -37,6 +37,11 @@
                   :isForm="false" 
                   :projectScope="docData.projectOverview.scope"
                 />
+
+                <RequestingServiceAgency 
+                :isForm="false"
+                :selectedServiceOrAgency="docData.organization.service_agency"
+                />
```

Comment:
In preview, this needs to be here as in the Figma
![Screen Shot 2022-09-23 at 11 09 30 AM](https://user-images.githubusercontent.com/90333349/191994046-c1ed1d88-6813-460b-9bdb-b8537d1631de.png)